assets/sass/components/global/_googlesitekit-button-icon.scss

```diff
@@ -35,12 +35,24 @@
 
 	&.googlesitekit-button-icon--spinner__running {
 
-		.mdc-button__label {
-			margin-right: 8px;
-		}
-
 		svg circle {
 			stroke: $c-white;
 		}
+
+		&.googlesitekit-button-icon--spinner__before {
+
+			.mdc-button__label {
+				margin-inline-start: 8px;
+				margin-left: 8px;
+			}
+		}
+
+		&.googlesitekit-button-icon--spinner__after {
+
+			.mdc-button__label {
+				margin-inline-end: 8px;
+				margin-right: 8px;
+			}
+		}
```

Comment:
Hi, that's a great idea. I've used a combination of both `inline-start/end` and ~`body[dir="rtl"]`~ `html[dir="rtl"]` override which seems to work in all cases. I'm inclined to keep the `inline-start/end` so we don't forget about them. I've also added TODO comments explaining what is needed to be done once we upgrade our SASS version. Let me know what you think. :heart:

Reply:
Just noting that I'd initially incorrectly blame SASS for this, when it's actually PostCSS what's causing this. I've filled a follow up issue to modernize our CSS infrastructure to explore this further and updated the code comments accordingly. #8414 

Cheers!

Reply:
That's great, thanks @kuasha420. My mistake using `body` instead of `html`, thanks for spotting that. The implementation and comments LGTM and it's great you've created #8414 to upgrade PostCSS in due course.

Top stuff! I'll get this merged now. Cheers :)